functions-app/report-psychometrician/service/anomaly-report.service.js

```diff
@@ -351,9 +355,15 @@ anomalyReportService.detectQuestionsThatWereShownForTooLong = (check) => {
   const audits = anomalyReportService.filterAllRealQuestionsAndPauseAudits(check)
   const config = check.data.config
   const head = R.head(audits)
+  if (!head) {
```

Comment:
From what I see you return when there's no single audit record.

Is that being handled properly afterwards when you return ?

Reply:
Yes.

We bail out of this test and move on to the next.